split/README.md

```diff
@@ -24,6 +30,35 @@ Integrate Split with Datadog to:
 
 Split data should be flowing into Datadog.
 
+### RUM Feature Flag Tracking
```

Comment:
```suggestion
### Feature Flag Tracking integration
```

Reply:
Suggesting this for consistency with other integrations being updated -- feel free to update #1768 and #1770 with `RUM feature flag tracking` if you prefer